<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parameter2</class>
 <widget class="QWidget" name="parameter2">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="comfirm_Button">
      <property name="text">
       <string>确认修改</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="return_Button">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="board_name_groupBox">
      <property name="title">
       <string>名称设置</string>
      </property>
      <widget class="QLabel" name="board_name_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>工装板名称：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="boardname">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>461</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="par_tips">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Ubuntu&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="wifigroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>wifi设置</string>
   </property>
   <widget class="QComboBox" name="wifissid">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>***</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>***</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="wifissidlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SSID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="wifikeylabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>KEY :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wifikey">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>***</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanpushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="parameter2_title">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;参数设置2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
